.github/workflows/release.yml

```diff
@@ -42,13 +42,24 @@ jobs:
           echo "GIT_TAG_NAME=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
           echo "Current version: $(git describe)"
 
+      - name: Cache Buildroot
+        id: restore-cache-br
+        uses: actions/cache@v2
+        with:
+          path: buildroot
+          # Stupid hack to invalidate cache from UI
+          # https://github.com/actions/cache/issues/2#issuecomment-673493515
+          key: br-${{ env.BUILDROOT_DIR }}-${{ env.BUILDROOT_VERSION }}-${{ secrets.BR_CACHE_VERSION }}
+
       - name: Download Buildroot
+        if: steps.restore-cache-br.outputs.cache-hit != 'true'
         run: |
           wget -qc https://buildroot.org/downloads/buildroot-${{ env.BUILDROOT_VERSION }}.tar.gz
           tar -zxf buildroot-${{ env.BUILDROOT_VERSION }}.tar.gz
           mv buildroot-${{ env.BUILDROOT_VERSION }} ${{ env.BUILDROOT_DIR }}
 
       - name: Cache Build
+        id: restore-cache-output
         uses: actions/cache@v2
         with:
           path: output
```

Comment:
it would be easier to just add the buildroot path here :) that way we don't have to work with two cache files

Reply:
Good point, trying again with your suggestion!